src/main/java/seedu/duke/commands/ShowRateCommand.java

```diff
@@ -59,18 +59,19 @@ public void execute(Ui ui, AccountList accounts) {
     }
 
     /**
-    * Prints the exchange rate between two currencies with a specified amount
-    * @param instance a Forex object containing the exchange rate
-    * @param amt a float of the amount to be converted on the exchange rate
-    * @return a string containing the exchange rates to be printed
-    * @throws InvalidNumberException if the amount is negative
-    */
-    private String getRateString(Forex instance, float amt) throws InvalidNumberException {
+     * Prints the exchange rate between two currencies with a specified amount
+     *
+     * @param instance a Forex object containing the exchange rate
+     * @param amt      a float of the amount to be converted on the exchange rate
+     * @return a string containing the exchange rates to be printed
+     * @throws InvalidNumberException if the amount is negative
+     */
+    private String getRateString (Forex instance, float amt) throws InvalidNumberException {
         if (amt < 0) {
             throw new InvalidNumberException();
         }
         String from = Account.currencyToString(instance.getInitial());
         String to = Account.currencyToString(instance.getTarget());
-        return String.format("%.2f %s = %.2f %s", amt, from, instance.convert(amt), to);
+        return String.format("%.2f %s = %,10.6f %s", amt, from, instance.convert(amt), to);
```

Comment:
Good use of %, to present large numbers more readably. Should add this same small simple change to other command outputs as well